staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -96,6 +96,12 @@ const (
 	// This feature is currently PRE-ALPHA and MUST NOT be enabled outside of integration tests.
 	TestOnlyCBORServingAndStorage featuregate.Feature = "TestOnlyCBORServingAndStorage"
 
+	// owner: @serathius
+	// beta: v1.32
```

Comment:
We no longer add information about stages here, as this is now easy to get from VersionedSpec below.

But please add a link to kep.

Reply:
There is no KEP, got agreement from api machinery leads to move forward with btree without KEP until we want to extend caching to more API surface.